core/client/fs/src/main/java/alluxio/client/file/FileSystemUtils.java

```diff
@@ -39,6 +40,14 @@ public final class FileSystemUtils {
   // prevent instantiation
   private FileSystemUtils() {}
 
+  /**
+   * @param conf the configuration to get info from
+   * @return whether metadata cache is enabled in the given configuration
+   */
+  public static boolean metadataEnabled(AlluxioConfiguration conf) {
```

Comment:
so far there are several caches, could the name be changed more explicit to indicate the fuse user space cache?

Reply:
There are two metadata cache
- FUSE KERNEL metadata cache enabled by mount option (e.g. -o kernel_cache)
- USER SPACE metadata cache, currently we only have client side metadata cache, defined by this property key.
It's shared by all clients, not just fuse clients, e.g. S3 API can use it, java API can use it, not just POSIX API